beads_web/src/main/java/com/beads/web/vaadin/view/order/component/OrderTableModel.java

```diff
@@ -37,7 +39,9 @@ private void init() {
   }
 
   private void populateContainerFullList() {
-    populateContainer(orderDao.getAllOrder());
+    SearchCriteria searchCriteria = new SearchCriteria();
+    searchCriteria.setDateOfOrder(LocalDateTime.now());
+    populateContainer(orderDao.getOrdersBySearchCriteria(searchCriteria));
```

Comment:
this line can be replace with invocation of 
`fireTableDataChange(searchCriteria)`